### ISSUE TITLE
Intune connector prerequisites

### ISSUE THREAD

**Issue by ncheymol** [MicrosoftDocs/memdocs#2807]
The intune connector prerequisites are diluted in the installation steps which makes it easy to miss some of them. 

Could you add a prerequisite section with OS, compatible proxy types and the need of a Licence on the intune or global administrator ?

(admin edit: update doc details with latest)

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 3f93dd93-e003-36fc-02d6-07166537d914
* Version Independent ID: 1b860d13-bcf5-50da-89f9-ee12ece6edb2
* Content: [Enrollment for hybrid Azure AD-joined devices - Windows Autopilot](https://learn.microsoft.com/en-us/mem/autopilot/windows-autopilot-hybrid)
* Content Source: [memdocs/autopilot/windows-autopilot-hybrid.md](https://github.com/MicrosoftDocs/memdocs/blob/main/memdocs/autopilot/windows-autopilot-hybrid.md)
* Product: **windows-client**
* Technology: **itpro-deploy**
* GitHub Login: @frankroj
* Microsoft Alias: **frankroj**

**Comment by yogkumgit** [comment]
@msbemba

**Comment by msbemba** [comment]
@ncheymol Thank you for your feedback. We have created a new section for pre requisites via PR#https://github.com/MicrosoftDocs/memdocs/pull/2841

Once the author approves, the changes will be live.

Thanks
Sri

**Comment by frankroj** [comment]
Duplicating prerequisites in two parts of the article was not a good idea. Instead, I broke the prerequisites section into two sections - client prerequisites and Intune connector server prerequisites, and then listed all prerequisites under the Prerequisites section. Down in the Install Intune connector section, I then added a link back Intune connector server prerequisites. This was done via PR https://github.com/MicrosoftDocs/memdocs-pr/pull/9501
